architecture/index.md

```diff
@@ -0,0 +1,31 @@
+# Architecture overiew
+
+## How data flows through the CDapp
+
+In principle, we want to handle all interactions with the chain outside of the client. We do this using the block ingestor and lambda functions.
+
+### The Block Ingestor
```

Comment:
It needs a section on "why' we chose this approach.

TL;DR; Because we want to have a centralized (ie: faster) API, but we also want the data to be read-only, meaning the chain is the source of truth